### PR TITLE
chore(docker): add image that contains gcloud and aws

### PR DESCRIPTION
**Pull request by solomonope**
<!--  Thanks for sending a pull request!  Here are some tips for you:

**What this PR does / why we need it**:
This PR creates an image that include gcloud, awscli and gardencli in one image
**Which issue(s) this PR fixes**:
It is meant to be used for gardencloud staging deployment
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: garden-service/bin/build-containers.sh [hunk 1]
```diff
@@ -10,6 +10,7 @@ version=${args[0]:-$(git rev-parse --short HEAD)}
 base_tag=gardendev/garden:${version}
 aws_tag=gardendev/garden-aws:${version}
 gcloud_tag=gardendev/garden-gcloud:${version}
+all_tag=gardendev/garden-all:${version}
```
Comment (eysi09) [inline garden-service/bin/build-containers.sh:13]: Can we find a better name than `garden-all`? It's not clear to the end user what `all` means in this context? What about `garden-all-providers`, `garden-all-clouds`, or simply `garden-aws-gcloud`. 

I think I actually like the last one best (if we don't foresee adding more credential helpers) but open to suggestions.

Comment (solomonope) [reply garden-service/bin/build-containers.sh]: I have updated it, I generally don't like length names, for me a short name and a bit of context works.

Comment (eysi09) [reply garden-service/bin/build-containers.sh]: Yeah, that's valid point as well. I'll approve this for now and we can later make the changes mentioned internally.

**Comment by edvald** [comment]
Merging this, but wanted to mention that we do plan on changing our docker image structure, and plan on merging all the individual provider-specific ones into a single one and/or making different version tags within a single repo, instead of all the specific repos.

**Pull request merged**